src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int ERR_OUT_OF_BOUNDS = -2;
+    private static final int ERR_WRONG_FORMAT = -1;
+    private static final int INVALID_DONE = -1;
+    private static final int EXIT = 0;
+    private static final int LIST = 1;
+    private static final int MARK = 2;
+    private static final int ADD = 3;
+    private static final int ADD_TODO = 4;
+    private static final int ADD_DEADLINE = 5;
+    private static final int ADD_EVENT = 6;
+
+    public static Task[] Tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        loopCommands();
+        sayGoodbye();
```

Comment:
Nice clean main method.

Reply:
Agree. It shows exactly what the program would do at a top level without clutter. However I think that loopCommands() method could be named better. Could use a while loop with executeCommand() and include the getCommand() method here instead. This is to show that the program prints a welcome message, takes user input, executes some commands from the user input, then exit after loop is escaped. Instead of just looping some commands with loopCommands().